core-plugins/src/main/java/co/cask/hydrator/plugin/batch/source/AzureBatchSource.java

```diff
@@ -61,7 +61,6 @@ public AzureBatchSource(AzureBatchConfig config) {
     @Override
     protected Map<String, String> getFileSystemProperties() {
       Map<String, String> properties = new HashMap<>(super.getFileSystemProperties());
-      properties.put("fs.defaultFS", String.format("wasb://%s@%s/", container, account));
```

Comment:
I was just saying we should remove 'container' as a property from the PluginConfig if it is no longer being used.

If the path always needs to be of the form 'wasb://<container>@<account>/...', then I think what you mentioned about prefixing makes sense. If we do that, should also update the documentation, and add a validation check to make sure it's just a path and not prefixed with 'wasb...'